website/src/styles/main.css

```
@@ -85,12 +85,62 @@ body {
+/* Sticky category quick-nav: icon + live-count jump chips above the card grid.
+   A single horizontally scrollable row keeps the sticky bar short. Plain
+   in-page anchors, so it works without JavaScript. The category name is the
+   accessible label (title + aria-label); the icon is decorative. */
+.category-nav {
+  position: sticky;
+  top: 0;
+  z-index: 30;
+  /* Hidden on phones (search + role filter cover navigation there, and the
+     sticky strip would eat scarce vertical space); shown from sm upward. */
+  @apply hidden sm:block mb-8 py-2;
+  background: var(--color-bg);
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)